lib/matplotlib/backends/backend_ps.py

```diff
@@ -867,9 +867,6 @@ def _print_figure(
         # find the appropriate papertype
         width, height = self.figure.get_size_inches()
         if papertype == 'auto':
-            _api.warn_deprecated("3.8", name="papertype='auto'",
-                                 addendum="Pass an explicit paper type, 'figure', or "
-                                 "omit the *papertype* argument entirely.")
             papertype = _get_papertype(*orientation.swap_if_landscape((width, height)))
```

Comment:
I would leave a comment about this here, or it might be forgotten when removing the deprecation in 3.10.

Reply:
And/or add a comment to the new deprecation site, because that’s where we look at during the removal.

Reply:
Sure, done.